test/e2e/specs/wp-my-home-support-search-spec.js

```diff
@@ -128,19 +122,34 @@ describe( `[${ host }] My Home "Get help" support search card: (${ screenSize })
 			const invalidSearchQueryReturningNoResults = ';;;ppp;;;';
 
 			await supportSearchComponent.searchFor( invalidSearchQueryReturningNoResults );
-			const resultsCount = await supportSearchComponent.getSearchResultsCount();
+
+			const searchResults = await supportSearchComponent.waitForSearchResultsNotToBePresent();
+
+			const resultsCount = await supportSearchComponent.getErrorResultsCount();
 
 			const hasNoResultsMessage = await supportSearchComponent.hasNoResultsMessage();
 
+			assert.equal( searchResults, true, 'The API Search Results are not displayed.' );
+
 			assert.equal( hasNoResultsMessage, true, 'The "No results" message was not displayed.' );
 
-			assert.equal( resultsCount, 5, 'There are no contextual results displayed.' );
+			assert.equal( resultsCount, 5, 'The default error results are displayed.' );
```

Comment:
Again I'm not sure the error message describes what went _wrong_ well enough unless you have a full context and understanding of how the support search works (like you do).

Reply:
All assertions are now written with the perspective of, "How would this be most helpful if you encounter the error."